webpack.mix.js

```diff
@@ -0,0 +1,86 @@
+const CleanWebpackPlugin = require('clean-webpack-plugin');
+const CopyWebpackPlugin = require('copy-webpack-plugin');
+const mix = require('laravel-mix');
+const { escapeRegExp } = require('lodash');
+const { sep } = require('path');
+
+/*
+ |--------------------------------------------------------------------------
+ | Mix Asset Management
+ |--------------------------------------------------------------------------
+ |
+ | Mix provides a clean, fluent API for defining some Webpack build steps
+ | for your Laravel application. By default, we are compiling the Sass
+ | file for the application as well as bundling up all the JS files.
+ |
+ */
+
+mix.setPublicPath('../glutenblocks');
+
+mix
+    .options({
+        processCssUrls: false,
+        lang: 'sass',
+    })
+    .webpackConfig({
+        devtool: '#source-map',
+        output: {
+            chunkFilename: 'build/[name].js?id=[chunkhash]',
```

Comment:
It is really better not to use a query parameter for cache busting.

```suggestion
            chunkFilename: 'build/[name].[chunkhash].js',
```

Reply:
The suggestion was implemented